setup/aix61/manualBootstrap.md

```diff
@@ -85,60 +122,59 @@ ln -s /opt/freeware/bin/ccache/ccache /opt/freeware/bin/ccache/g++
 ln -s /opt/freeware/bin/ccache/ccache /opt/freeware/bin/ccache/gcc
 cd ../..
 rm -rf tmp
+```
+
+## Install svn
 
-5) install svn, only needed for v8 testing
+*N.B. Only needed for v8 testing*
```

Comment:
@mhdawson do we still need to install svn? I thought we'd moved to git for the v8 testing. If so I'll remove this as well.